### PR TITLE
Remove mocha scripts from CONTRIBUTING.md and add test:inspect script

### PR DESCRIPTION
**Pull request by invokesus**

### CODE REVIEW

**Comment by kanitw** [comment]
LGTM - let's see what @domoritz  too.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ If you only want subset of these actions, you can use:
 
 #### Fast iteration testing
 
-To quickly run tests without long compile times, run `yarn tsc -w` in a separate terminal session. Then run `yarn mocha:test` to quickly run tests (or `yarn mocha:test --inspect --debug-brk` to inspect tests). Please note that this only runs unit tests; you should run the full tests before committing code.
+To quickly run tests without long compile times, run `yarn tsc -w` in a separate terminal session. Then run `yarn jest` to quickly run tests (or `yarn test:inspect` to inspect tests). Please note that this only runs unit tests; you should run the full tests before committing code.
```
Comment (domoritz) [inline CONTRIBUTING.md]: You don't actually need to run `yarn tsc` in order to run the tests. Can you remove that?

File: package.json [hunk 2]
```diff
@@ -47,6 +47,7 @@
     "site": "bundle exec jekyll serve --incremental",
     "lint": "tslint -p .",
     "test": "jest test/ && npm run lint && npm run schema && jest examples/ && npm run test:runtime",
+    "test:inspect" :"node --inspect-brk ./node_modules/.bin/jest --runInBand test",
```
Comment (domoritz) [inline package.json]: Doesn't jest have a shortcut for this?

Comment (domoritz) [reply package.json]: I couldn't find anything so I guess this is good.

File: CONTRIBUTING.md (hunk 1 above)
Comment (invokesus) [reply CONTRIBUTING.md]: Done.

File: package.json (hunk 2 above)
Comment (invokesus) [reply package.json]: Got this from the [jest docs](https://facebook.github.io/jest/docs/en/troubleshooting.html).

**Pull request merged**